include/IMeshBuffer.h

```diff
@@ -21,7 +21,7 @@ namespace irr
 {
 namespace scene
 {
-
+	//! Available vertex attribute ids
```

Comment:
could add that As of 2018 most OpenGL implementations support 16 attributes (some CAD GPUs more)